Tests/LinuxMain.swift

```diff
@@ -1,7 +1,13 @@
 import XCTest
+import Quick
 
-import SwerverTests
+@testable import SwerverTests
 
-var tests = [XCTestCaseEntry]()
-tests += SwerverTests.allTests()
-XCTMain(tests)
\ No newline at end of file
+QCKMain([
```

Comment:
Is there any way to automatically include all test files in here? This seems error prone and costly to maintain.